src/EventStore.Core/TransactionLog/Checkpoint/CheckpointMetric.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+
+namespace EventStore.Core.TransactionLog.Checkpoint {
+	// For now reading 'nonflushed' because the flushed values may give the impression
+	// of the checkpoints being further behind than they really are.
+	public class CheckpointMetric {
+		private readonly IReadOnlyCheckpoint[] _checkpoints;
+		private readonly Measurement<long>[] _measurements;
+		private readonly KeyValuePair<string, object>[][] _tagss;
```

Comment:
typo?

Reply:
not a typo, just a somewhat clumsy name. each element in the array is an array of tags